validphys2/src/validphys/datafiles/new_commondata/LHCB_Z0J_13TEV_2022/metadata.yaml

```diff
@@ -50,18 +51,19 @@ implemented_observables:
                   label: '$\frac{d^2\sigma}{d p_T^Z y^Z}$',
                   units: "[pb]"}
     observable_name: DIMUON-PT-Y
-    process_type: EWK_PTRAP
+    process_type: _
```

Comment:
Here I left blank, because we are not going to use the double differential distribution. But if you wish I can implement the relative process type anyway.

Reply:
Why are we not going to use it?

(if we know already that we are not going to use it... why do we want to have it implemented?)

Reply:
Do you mean removing this distribution from the metadata? Observe that this part is not outputted in the yaml files exactly for that reason.

Reply:
yes, and don't include the kinematics/data/uncertainties files (which at least in the metadata are referenced)